### PR TITLE
Travis fixes (container infra)

### PR DESCRIPTION
**Pull request by damacus**
Remove gitter, as we're using slack
foodcritic fixes using sous chefs rules

### CODE REVIEW

**Comment by damacus** [comment]
@shortdudey123 regression....

File: .travis.yml [hunk 1]
```diff
@@ -53,12 +45,13 @@ env:
 before_script:
   - sudo iptables -L DOCKER || ( echo "DOCKER iptables chain missing" ; sudo iptables -N DOCKER )
   - eval "$(/opt/chefdk/bin/chef shell-init bash)"
+  - chef gem install kitchen-dokken
```
Comment (shortdudey123) [inline .travis.yml]: isn't this part of chefdk?

Comment (damacus) [inline .travis.yml]: yeah but we needed an updated version, as ChefDk released a few days before some container fixes went in.

Comment (shortdudey123) [inline .travis.yml]: ah

**Comment by shortdudey123** [comment]
👍

**Pull request merged**